bin/alluxio-monitor.sh

```diff
@@ -209,14 +209,18 @@ main() {
   case "${ACTION}" in
     all)
       prepare_monitor "Starting to monitor ${CYAN}all remote${NC} services."
-      run_monitors "master" "${HOSTS}" "${MODE}"
-      run_monitors "worker" "${HOSTS}" "${MODE}"
-      run_monitors "proxy" "${HOSTS}" "${MODE}"
+      run_monitors "masters" "${HOSTS}" "${MODE}"
+      run_monitors "job_masters" "${HOSTS}" "${MODE}"
+      run_monitors "workers" "${HOSTS}" "${MODE}"
+      run_monitors "job_workers" "${HOSTS}" "${MODE}"
+      run_monitors "proxies" "${HOSTS}" "${MODE}"
```

Comment:
this code looks strange - isn't the set of hosts different for workers vs masters vs proxies? But they all use `${HOSTS}` here.

Reply:
When passing arguments to `run_monitors`, an empty 2nd argument will result in pulling the the node list from the respective file  under the conf directory (See https://github.com/Alluxio/alluxio/blob/e52d802d352802fcf378017158d677c79b34b87b/bin/alluxio-monitor.sh#L124-L149). I will update these to pass `""` instead so that the node list will be pulled from the configuration files.

Reply:
After doing some more digging - we actually don't need to change it from `${HOSTS}` to `""` because when `alluxio-start.sh` calls `alluxio-monitor.sh` it will only ever populate the `${HOSTS}` argument if `alluxio-start.sh` is restarting the workers, otherwise it is always an empty string.